sherpa-onnx/csrc/offline-recognizer-transducer-impl.h

```diff
@@ -117,7 +128,8 @@ class OfflineRecognizerTransducerImpl : public OfflineRecognizerImpl {
                                -23.025850929940457f);
 
     auto t = model_->RunEncoder(std::move(x), std::move(x_length));
-    auto results = decoder_->Decode(std::move(t.first), std::move(t.second));
+    auto results =
+        decoder_->Decode(std::move(t.first), std::move(t.second), ss, n);
```

Comment:
I suggest we pass a nullptr instead of ss when context graph is empty. Otherwise, it does extra work in modified beam search.

Reply:
I think pass ss is OK, because some streams would have context graph, some streams would not, we will need a loop to get the context graph for each stream anyway.

Reply:
in that case, I suggest putting the loop outside of `Decode()`.
Otherwise, you need to check for each frame inside `Decode()`.

If there are no streams requiring that, we don't need to do extra work for each frame inside `Decode()`.

Reply:
I just revisited the code, it seems make no difference to add a extra loop outside of `Decode()`,  the loop inside `Decode()` is for the whole sequence not each frame.  Shall we keep it as it is?

Reply:
Ok, but for the online stream, I suggest putting it outside of `Decode()`.